packages/node-runtime-worker-thread/src/worker-thread-evaluation-listener.ts

```diff
@@ -56,6 +59,7 @@ export class WorkerThreadEvaluationListener {
             (Promise.resolve() as Promise<never>)
           );
         },
+        version: workerRuntime.evaluationListener?.version,
```

Comment:
How do we provide this value in non-CLI products (vscode/Compass/IntelliJ)? I do think that's something we're going to have to figure out, and unfortunately I don't have an obvious answer for it myself

Reply:
if I understand correctly, making it a generated constant inside `shell-api` essentially handles that right?